projects/batfish-client/src/main/java/org/batfish/client/BfClientRequestFilter.java

```diff
@@ -0,0 +1,19 @@
+package org.batfish.client;
+
+import javax.ws.rs.client.ClientRequestContext;
+import javax.ws.rs.client.ClientRequestFilter;
+import javax.ws.rs.ext.Provider;
+import org.batfish.common.CoordConsts;
+import org.batfish.common.Version;
+
+/** This filter adds default authentication information (apikey and version) to HTTP headers for
+ * every HTTP request. */
+@Provider
+public class BfClientRequestFilter implements ClientRequestFilter {
```

Comment:
Why doesn't this use the client's API key?

This seems somehow like the wrong approach to me.

Reply:
Pick a class name that reflects what this filter does.